src/app/components/navbar/navbar.component.html

```diff
@@ -1,23 +1,17 @@
-<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+<nav class="cnavbar navbar-expand-lg navbar-dark">
 	<div class="container-fluid">
 		<!-- Navbar Brand -->
-		<a class="navbar-brand" routerLink="/">MATBay!</a>
-
-		<!-- Collapse button-->
-		<button class="navbar-toggler" type="button" (click)="basicNav.toggle()" aria-expanded="false"
-			aria-label="Toggle navigation">
-			<i class="fas fa-bars"></i>
-		</button>
-		<div class="collapse navbar-collapse" id="navbarNav" mdbCollapse #basicNav="mdbCollapse">
-			<button mat-raised-button class="btn btn-primary purple dropdown-toggle navbar-button" [matMenuTriggerFor]="menu">
+		<a class="navbar-brand" routerLink="/">Matbay!</a>
+		<div class="container">
+			<div class="container left">
+			<button mat-raised-button class="btn btn-primary purple navbar-button" [matMenuTriggerFor]="menu">
```

Comment:
Il faut garder l'aspect dropdown button pour savoir que ce bouton est une liste
```suggestion
			<button mat-raised-button class="btn btn-primary dropdown-toggle purple navbar-button" [matMenuTriggerFor]="menu">
```